Workade/Views&ViewModels/Map/MapInfoView.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  MapInfoModalView.swift
+//  Workade
+//
+//  Created by Inho Choi on 2022/11/10.
+//
+
+import NMapsMap
+import UIKit
+
+class MapInfoView: UIView {
+    init() {
+        super.init(frame: .zero)
+        self.translatesAutoresizingMaskIntoConstraints = false
+        self.backgroundColor = .white
+        self.layer.cornerRadius = 30
+        
+        setLayout()
+    }
+    
+    var titleLabel: UILabel = {
+        let title = UILabel()
+        title.translatesAutoresizingMaskIntoConstraints = false
+        title.font = .customFont(for: .captionHeadlineNew)
+        title.textColor = .black
+        return title
+    }()
+    
+    var markerImage: UIImageView = {
+        let imageView = UIImageView()
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        return imageView
+    }()
+    
+    lazy var mapButton: UIButton = {
+        let mapButton = UIButton()
+        mapButton.translatesAutoresizingMaskIntoConstraints = false
+        mapButton.setImage(UIImage(named: "navermapbutton"), for: .normal)
+        mapButton.addAction(UIAction(title: "Go") { [weak self] _ in
+            self?.naverMapButtonTapped()
+        }, for: .touchUpInside)
+        
+        return mapButton
+    }()
+    
+    required init(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func setLayout() {
+        addSubview(titleLabel)
+        NSLayoutConstraint.activate([
+            titleLabel.topAnchor.constraint(equalTo: self.topAnchor, constant: 22),
+            titleLabel.bottomAnchor.constraint(equalTo: self.bottomAnchor, constant: -22),
+            titleLabel.leadingAnchor.constraint(equalTo: self.leadingAnchor, constant: 85),
+            titleLabel.trailingAnchor.constraint(equalTo: self.leadingAnchor, constant: 200 + 85)
+        ])
```

Comment:
[P1] Figma상의 Hug로 되어있기 때문에 유동적으로 변해야 하는게 맞는것 같습니다! 
leading 과 trailing을 85씩만 할당하면 기기 너비와 상관없이 유동적으로 양쪽 모두 85만큼만 떨어진 상태로 titleLabel의 너비가 동적으로 변할것 같아요! 

200 고정값 + 85를 하게되면 기기별 대응에서 취약할수 있을것 같습니다!

```suggestion
        NSLayoutConstraint.activate([
            titleLabel.topAnchor.constraint(equalTo: self.topAnchor, constant: 22),
            titleLabel.bottomAnchor.constraint(equalTo: self.bottomAnchor, constant: -22),
            titleLabel.leadingAnchor.constraint(equalTo: self.leadingAnchor, constant: 85),
            titleLabel.trailingAnchor.constraint(equalTo: self.leadingAnchor, constant: 85)
        ])
```

Reply:
이렇게 될 경우에 기기의 넓이가 0으로 변환됩니다.
차라리 trailingAnchor를 설정하지 않음으로 기기가 알아서 설정할 수 있도록 할까요?

Reply:
맨 마지막 줄을
titleLabel.trailingAnchor.constraint(equalTo: self.trailingAnchor, constant: -85)로 수정해보면 될 것 같습니당!

Reply:
엣지케이스일수도 있지만, trailingAnchor를 설정하지않을 경우 우측에 대한 제약조건이 없기 때문에 해당 장소(식당 등)의 이름이 굉장히 길 때, 레이블과 우측 맵 버튼이 겹치게 될 것입니다.